1. Select basics/Task 1 Select/Result/Task1_Natalli_Sholomitskaya.sql

```diff
@@ -0,0 +1,29 @@
+select * from employees;
```

Comment:
please add comment to each select with task number like -- Task 1 ... or /*Task 1 ....*/